istio-control/istio-autoinject/files/injection-template.yaml

```diff
@@ -85,9 +87,9 @@ template: |
     - --domain
     - $(POD_NAMESPACE).svc.{{ .Values.global.proxy.clusterDomain }}
     - --configPath
-    - "/etc/istio/proxy"
+    - "{{ .ProxyConfig.ConfigPath }}"
```

Comment:
I don't think either are needed or tested, was planning to deprecate the fields.